### PR TITLE
Add split_pdf_* parameters to all related code examples

### PR DESCRIPTION
**Pull request by Paul-Cornell**

### CODE REVIEW

File: api-reference/api-services/examples.mdx [hunk 1]
```diff
@@ -27,7 +27,10 @@ The `hi_res` strategy supports different models, and the default is `layout_v1.1
   -H 'unstructured-api-key: $UNSTRUCTURED_API_KEY' \
   -F 'files=@sample-docs/layout-parser-paper.pdf' \
   -F 'strategy=hi_res' \
-  -F 'hi_res_model_name=yolox'
+  -F 'hi_res_model_name=yolox' \
```
Comment (ninalopatina) [inline api-reference/api-services/examples.mdx:30]: do we want to update this to layout_v1.1.0 since this is now the preferred model to use?

**Review by ninalopatina — CHANGES_REQUESTED** [review]
just one note on the default hi_res model in the first file change

File: api-reference/api-services/examples.mdx (hunk 1 above)
Comment (MKhalusova) [reply api-reference/api-services/examples.mdx:30]: `layout_v1.1.0`. We can show one or the other throughout the documentation, but this comment is unrelated to this PR, we can discuss and address it separately.

**Review by MKhalusova — APPROVED** [review]

File: api-reference/api-services/examples.mdx (hunk 1 above)
Comment (MKhalusova) [reply api-reference/api-services/examples.mdx:30]: Please unblock the merge.

Comment (Paul-Cornell) [reply api-reference/api-services/examples.mdx]: Changed/updated.

**Pull request merged**